clojure/fira_code/main.clj

```
@@ -13,21 +13,59 @@
    [flatland.ordered.map :refer [ordered-map]]))
 
 
+(defn load-font [path]
+  "Loads the font object from the given file path, or returns nil if the file
+  cannot be read."
+  (try
+    (glyphs/load path)
+    (catch Exception _ nil)))
```

Why would you suppress exception like this? It is kind of important